webserver.py

```diff
@@ -1,11 +1,15 @@
 #!/usr/bin/env python3
 import sqlite3
 from datetime import datetime, timedelta
-import time
+from json import JSONDecodeError
 
 import pytz
 from flask import Flask, g, render_template, request, redirect
 
+import urllib.request
+import json
+import time
```

Comment:
Probably a good idea to order these imports in a standard way:
```suggestion
from datetime import datetime, timedelta
import json
from json import JSONDecodeError
import time
import urllib.request

from flask import Flask, g, render_template, request, redirect
import pytz
import sqlite3

from common import mac_int_to_str, mac_str_to_int
```
The general rule (from https://www.python.org/dev/peps/pep-0008/#imports ) is:

1. Standard library imports.
2. Related third party imports.
3. Local application/library specific imports.